alfresco-transform-libreoffice/alfresco-transform-libreoffice/src/main/java/org/alfresco/transformer/executors/JodConverterSharedInstance.java

```diff
@@ -26,22 +26,21 @@
  */
 package org.alfresco.transformer.executors;
 
-import static java.util.Arrays.asList;
-import static java.util.Objects.requireNonNull;
+import org.artofsolving.jodconverter.office.DefaultOfficeManagerConfiguration;
```

Comment:
Minor: format only change ? Do we need team standard for IDE imports to reduce flip-flop when editing files ?

Reply:
Agreed, changes here are not needed. Reverted to original.